src/category/index.ts

```diff
@@ -0,0 +1,30 @@
+export interface CategoryLevel {
+  id: number
+  uuid: string
+  index: number
+  props: string
+}
+
+export interface CategoryProps {
+  name: string
+}
+
+export interface CategoryLevels {
+  [key: number]: Category
```

Comment:
Category vs CategoryLevel

Reply:
I wonder if we need CategoryLevel at all? Isnt it a list of categories?

Reply:
Category and CategoryLevel are 2 different things, in this case you have to use CategoryLevel